code/datums/components/gunpoint.dm

```diff
@@ -169,6 +185,7 @@
 	shooter.visible_message(span_danger("[shooter] breaks [shooter.p_their()] aim on [target]!"), \
 		span_danger("You are no longer aiming [weapon] at [target]."), ignored_mobs = target)
 	to_chat(target, span_userdanger("[shooter] breaks [shooter.p_their()] aim on you!"))
+	remove_indicator()
 	qdel(src)
```

Comment:
Why do we need to call `remove_indicator()` every time instead of doing it on the component's destroy? This is too fragile

Reply:
Destroy() proc, right?

Reply:
Ya